models/product-projection-alternative/src/presets/happy-cow-milk-product-projection.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  LocalizedField,
+  LocalizedString,
+} from '@commercetools-test-data/commons';
+import { TSpecializedBuilder } from '@commercetools-test-data/core';
+import { ProductVariant } from '@commercetools-test-data/product';
+import { ProductType } from '@commercetools-test-data/product-type';
+import { ProductProjectionGraphql, ProductProjectionRest } from '..';
+import { TProductProjectionGraphql, TProductProjectionRest } from '../types';
+
+// TODO: This should be moved to commons package
+const createLocalizedFields = (params: [string, string][]) => {
+  return params.map(([locale, value]) =>
+    LocalizedField.random().locale(locale).value(value)
+  );
+};
+
+export const happyCowMilkRest =
+  (): TSpecializedBuilder<TProductProjectionRest> => {
+    const productName = LocalizedString.presets
+      .empty()
+      .en('Happy Cow Milk')
+      .de('Fröhliche Kuhmilch');
+
+    const productDescription = LocalizedString.presets
+      .empty()
+      .en('Very happy milk produced by very happy cow!')
+      .de('Sehr glückliche Milch von sehr glücklicher Kuh!');
+
+    const slug = LocalizedString.presets.empty().en('happy-cow-milk-slug');
+
+    return ProductProjectionRest.random()
+      .productType(ProductType.presets.milk())
```

Comment:
This is initially where my will to share code triggers. Rightfully or not. This will "blow" presets up into two variations from one, right?

Reply:
Yes. With this approach we will need to duplicate the presets for the models where the presets themselves populate model properties which have different type across the two representations.

Reply:
Could be maybe solved though if we could e.g. hydrate or seed a builder with a format which works for either or. Throwing 🍝 at the wall but let's assume we can have where we say `[ ['productType', ProductType.presets.milk()], ['slug', 'happy-cow-milk-slug'] ]` in tuples like `Object.fromObject` we could share the commonality between builder types and reuse it. 

Sort of wondering if that would remove the whole proxy in the end. A bit too wide of a scope I suppose.